apps/frontend/package.json

```diff
@@ -115,6 +117,7 @@
     "vuex": "^3.1.2",
     "vuex-module-decorators": "^1.0.1",
     "xlsx": "^0.17.0",
+    "xlsx-populate": "^1.21.0",
```

Comment:
This dependency adds about 2MB to the version of the app that is sent to users, is there a reason we need 2 xlsx libraries?

![Screen Shot 2021-07-01 at 12 45 04 PM](https://user-images.githubusercontent.com/2308869/124162082-1e068900-da6c-11eb-9ea0-d28426f751a6.png)
![Screen Shot 2021-07-01 at 12 58 32 PM](https://user-images.githubusercontent.com/2308869/124162083-1e9f1f80-da6c-11eb-8296-3d9a5c7e76ac.png)